data/course-data/astr/courses.json

```diff
@@ -0,0 +1,274 @@
+{
+    "courses": [
+        {
+            "institution": "TOWSN",
+            "acadCareer": "UGRD",
+            "subject": "ASTR",
+            "catalogNumber": "161",
+            "courseTitle": "THE SKY AND THE SOLAR SYSTEM",
+            "termsOffered": "Fall, Spring & Summer",
```

Comment:
Ideally, these should be arrays of strings. If I had the more time, I was going to redesign our openai service to send OpenAI another request to accommodate for courses in the wrong semesters. Obviously, I never go to so this isn't current concern, but I'm just letting you know.